index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <title>Portfolio setup and mobile first</title>
+</head>
+<body>
+    <div>
+        <nav class="navbar">
+            <a class="box 1" href="https://www.github.com/AlejandraKeber">Alejandra</a>
```

Comment:
Your logo appear to be a little different from the given Figma design , Kindly fix it 👍 😄 
you can remove the underline in css by using `text-decoration: none`
![image](https://user-images.githubusercontent.com/87448628/185467664-df173a20-b187-4527-8538-f00e927d454e.png)